src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/media-edit-overlay/main.js

```diff
@@ -13,7 +13,12 @@
  * @constructor
  *
  **/
-define(['services/sulumedia/media-manager'], function(MediaManager) {
+define([
+    'services/sulumedia/media-manager',
+    'text!./info.html',
+    'text!./copyright.html',
+    'text!./versions.html'
+], function(MediaManager, InfoTemplate, CopyrightTemplate, VersionsTemplate) {
```

Comment:
These variables are not constructors, you should stick to the rules you have introduced by your own :wink: